fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -115,6 +128,7 @@ public String getLatestVersion(String url, Class<? extends Resource> resourceTyp
     public <T extends Resource> T getResource(String url, Class<T> resourceType) {
```

Comment:
Add documentation that says you can only call this for definitional resource types and that otherwise it will throw an IllegalArgumentException

Reply:
Added Javadoc